src/thunderbots/software/ai/world/ball.h

```diff
@@ -48,7 +51,26 @@ class Ball final
      */
     Vector velocity(double time_delta = 0.0) const;
```

Comment:
Some time stuff was added before, but it's a very light wrapper around std::chrono. See the `util/timestamps` file. The wrapper could be fleshed out a bit in order to provide better utility (I find std::chrono quite wordy to write).

I agree with your points about robustness.

Would you agree this could be left to a separate PR so that too many time/timestamp changes don't creep into this PR? I can make a ticket to track this in the meantime.

Reply:
I took a slightly better look at this, and although it seems quite easy to change this for these functions it would make sense as part of a larger change to our Timestamping system. I'd like to leave those changes separate since I feel that would get a bit out of scope.